src/main/java/shortestpath/DebugOverlayPanel.java

```diff
@@ -79,6 +86,12 @@ public Dimension render(Graphics2D graphics) {
         String time = String.format("%.2fms", milliTime);
         components.add(makeLine("Time:", time));
 
+        components.add(separator);
+
+        components.add(makeText("Path Owner:"));
+        String requester = currentParameters.getRequester().getFullyQualifiedName();
```

Comment:
Personally I prefer seeing `Shortest Path` and `Quest Helper` instead of `shortestpath.ShortestPathPlugin` and `questhelper.QuestHelperPlugin` here, so maybe change this to `getName()`? Plugin names are not required to be unique, but I think they all are.